js/core/tty/terminal.js

```diff
@@ -35,6 +35,9 @@ refresh();
 exports.color = vga.color;
 
 exports.print = function(text, repeat, fg, bg) {
+  // fix issue #53 where non-strings (ints, etc...) would not get printed
+  text = String(text);
```

Comment:
I think ```text = text.toString()``` is better, you are interested in the string literal, not a String object, and for string literals it will be returned itself instead of creating always an object...